projects/packages/forms/src/contact-form/class-contact-form-field.php

```diff
@@ -745,7 +745,7 @@ private function render_consent_field( $id, $class ) {
 	/**
 	 * Return the HTML for the multiple checkbox field.
 	 *
-	 * @param int    $id - the ID.
+	 * @param string $id - the ID (starts with 'g' - see constructor).
```

Comment:
$id is always a string. See contructor around line 150 where it is set. This is relevant because a valid html ID need to start with a letter, not a number.